specification/azurearcdata/resource-manager/Microsoft.AzureArcData/preview/2023-11-01-preview/examples/CreateOrUpdateArcSqlServerAvailabilityGroup.json

```diff
@@ -0,0 +1,259 @@
+                "backupPriority": 50,
+                "readOnlyRoutingUrl": null,
+                "readWriteRoutingUrl": null,
+                "seedingModeDescription": "AUTOMATIC"
+              },
+              "state": {
+                "availabilityGroupReplicaRole": "SECONDARY",
+                "operationalStateDescription": "ONLINE",
+                "recoveryHealthDescription": "ONLINE_IN_PROGRESS",
+                "synchronizationHealthDescription": "HEALTHY",
+                "connectedStateDescription": "CONNECTED",
+                "lastConnectErrorDescription": "",
+                "lastConnectErrorTimestamp": "2022-05-05T16:26:33.883Z"
+              }
+            }
+          ]
+        },
+        "databases": {
+          "value": [
+            {
+              "databaseName": "db1",
+              "replicaName": "testSqlServer\\INST1",
+              "isLocal": true,
+              "isPrimaryReplica": true,
+              "synchronizationStateDescription": "SYNCHRONIZED",
+              "isCommitParticipant": true,
+              "synchronizationHealthDescription": "HEALTHY",
+              "databaseStateDescription": "ONLINE",
+              "isSuspended": false
+            },
+            {
+              "databaseName": "db2",
+              "replicaName": "testSqlServer\\INST1",
+              "isLocal": true,
+              "isPrimaryReplica": true,
+              "synchronizationStateDescription": "SYNCHRONIZED",
+              "isCommitParticipant": true,
+              "synchronizationHealthDescription": "HEALTHY",
+              "databaseStateDescription": "ONLINE",
+              "isSuspended": false
+            }
+          ]
+        }
+      }
+    }
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "location": "southeastasia",
+        "id": "/subscriptions/00000000-1111-2222-3333-444444444444/resourceGroups/testrg/providers/Microsoft.AzureArcData/sqlServerInstances/testSqlServer_INST1/availabilityGroups/testAG",
+        "name": "testAG",
+        "type": "Microsoft.AzureArcData/sqlServerInstances/availabilityGroups",
+        "properties": {
+          "availabilityGroupId": "00000000-1111-2222-3333-444444444444",
+          "serverName": "testSqlServer",
+          "instanceName": "INST1",
+          "replicas": {
+            "value": [
+              {
+                "replicaId": "00000000-1111-2222-3333-444444444444",
+                "replicaName": "testSqlServerInstance",
+                "configure": {
+                  "endpointUrl": "TCP://mytest60-0.mytest60-svc:5022",
+                  "availabilityModeDescription": "SYNCHRONOUS_COMMIT",
+                  "failoverModeDescription": "EXTERNAL",
+                  "sessionTimeout": 10,
+                  "primaryRoleAllowConnectionsDescription": "ALL",
+                  "secondaryRoleAllowConnectionsDescription": "ALL",
+                  "replicaCreateDate": null,
+                  "replicaModifyDate": null,
```

Comment:
You shouldn't have null in examples unless the property is modeled as 'x-ms-nullable' true.

Also, its recommended NOT to model properties as 'x-ms-nullable' true, as a general rule.